deploy/lib/server_config.rb

```diff
@@ -2685,8 +2685,9 @@ def ServerConfig.properties(prop_file_location = @@path)
     properties = ServerConfig.load_properties(default_properties_file, "ml.")
     properties.merge!(ServerConfig.load_properties(properties_file, "ml."))
 
+    #Look for optional shared_config, if it is set grab the properties from path relative to the root of the roxy project
     if properties['ml.shared_config'] then
```

Comment:
@jmeekhof I'm totally good with this after speaking with you about the use case.